src/EditorFeatures/CSharpTest/ConvertToRecord/ConvertToRecordTests.cs

```diff
@@ -702,68 +773,56 @@ public class [|C|]
         public int P { get; init; }
         public bool B { get; init; }
 
-        public static bool operator ==(C? c1, C? c2) {
-            return c1!.Equals(c2);
-        }
-
-        public static bool operator !=(C? c1, C? c2) {
-            return !(c1 == c2);
+        public override bool Equals(object? other)
+        {
+            return other is C otherC && otherC.P == P && otherC.B == B && otherC.P == P;
         }
     }
 }
 ";
             var changedMarkup = @"
 namespace N
 {
-    public record C(int P, bool B);
+    public record C(int P, bool B)
+    {
+
```

Comment:
Blank line?

Reply:
Fixed by removing blank lines for the first member after we've done all our modifications